tests/vcr/tests.ouimeaux_device.test_bridge/test_light_turn_off.yaml

```diff
@@ -123,7 +123,7 @@ interactions:
 
       <u:SetDeviceStatus xmlns:u="urn:Belkin:service:bridge:1">
 
-      <DeviceStatusList>&lt;?xml version=&#x27;1.0&#x27; encoding=&#x27;utf-8&#x27;?&gt;
+      <DeviceStatusList>&lt;?xml version=&#x27;1.0&#x27; encoding=&#x27;UTF-8&#x27;?&gt;
```

Comment:
Just curious, is this an actual change from re-running vcr?  Why was it lowercase to start with?

Reply:
Yes, the tests were failing without this change. I'm not sure if this is related to #202 or or something else.

Reply:
ah, ya, that'd make sense, since the code actually does still have lowercase `uft-8` in it, so lxml must change it to uppercase.

Reply:
I did a git revert e375e12f4901461b02ceb08feb5c02a766c560ee and ran the test again. With #202 reverted I have to use lower-case 'utf'. So it seems that PR is where the change happened.

It doesn't seem to make a practical difference though. My bridge device works either way.

Reply:
But it sure does validate your reasoning for why we want to check the request body. It would have been nice to know about this change earlier.